### PR TITLE
Components

### PR DESCRIPTION
**Pull request by John-Arboleda**
# Initialize project with components
### Learning objectives
- Use React Router
- Use React Components
### Project requirements
- [x] Initialise React app.
- [x] The building blocks of your app should be set as re-usable components.
- [x] Add React Router and set <Route>s and <Link>s for the app's navigation

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import BookItem from './BookItem';
+import NewBook from './NewBook';
+
+function Books() {
+  return (
+    <div>
+      <BookItem />
```
Comment (afanimartin) [inline src/components/Books.js]: - Please render the `BookItem` in a list. Meaning, the `BookItem` should be wrapped in a list item `li`.

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Categories() {
+  return (
+    <div>
+      <h1>Under construction</h1>
```
Comment (afanimartin) [inline src/components/Categories.js]: - Please change this to a button with the text `Check status`. This sets up the project nicely for upcoming changes.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @John-Arboleda, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by keroloussamy — APPROVED** [review]
Hi @John-Arboleda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**